allennlp_models/generation/models/simple_seq2seq.py

```diff
@@ -162,18 +178,19 @@ def take_step(
             the source mask, and the decoder hidden state and context. Each of these
             tensors has shape `(group_size, *)`, where `*` can be any other number
             of dimensions.
+<<<<<<< HEAD
+=======
         step : `int`
             The time step in beam search decoding.
 
+>>>>>>> 5d9098f6084a12da77b02d40e0d9392113aeb805
```

Comment:
You checked in some unmerged files. I don't think it's serious, but we can't merge it like this.

Reply:
fixed